UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common/RadioNavigationalWarningsContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.EntityFrameworkCore;
+using UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning.DTO;
+
+namespace UKHO.MaritimeSafetyInformation.Common
+{
+    public class RadioNavigationalWarningsContext : DbContext
```

Comment:
I think DbContext can be excluded from code coverage.

Reply:
excluded from code coverage.